.env.development

```diff
@@ -0,0 +1,3 @@
+GATSBY_STRAPI_USER=techConnectAPI
+GATSBY_STRAPI_PW=^Xx1pr!C3di7
```

Comment:
Woa woa woa buddy, add this file to .gitignore ASAP

Also, if your dev builds are dependent on any environment variables, be sure to mention that in the changelog. If they need to be updated remotely, that needs to be know before merging changes.